client/packages/inventory/src/Stocktake/DetailView/modal/StocktakeLineEdit/StocktakeLineEditTables.tsx

```diff
@@ -117,10 +117,18 @@ export const BatchTable: FC<StocktakeLineEditTableProps> = ({
     [
       expiryDateColumn,
       {
-        width: 100,
+        width: 120,
         setter: patch => update({ ...patch, countThisLine: true }),
       },
     ],
+    {
```

Comment:
I was thinking to put it on the 'location' tab, since that only had one thing on it. I don't know if people will find it on that tab.. but I thought that mostly people won't need a comment anyway 🤷 
also the current comment field is very short, it could be longer on the location maybe?